modules/util/buildhelpers.py

```diff
@@ -976,4 +976,67 @@ def move_docs(module_docs_path):
+    "Firewall":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/firewall.yml",
+    "Firmware":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/firmware.yml",
+    "Group":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/group.yml",
+    "Image":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/image.yml",
+    "Instance":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/instance.yml",
+    "Kernel":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/kernel.yml",
+    "Logon Session":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/logon_session.yml",
+    "Module":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/module.yml",
+    "Named Pipe":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/named_pipe.yml",
+    "Network Share":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/network_share.yml",
+    "Network Traffic":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/network_traffic.yml",
+    "Pod":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/pod.yml",
+    "Process":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/process.yml",
+    "Scheduled Job":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/scheduled_job.yml",
+    "Script":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/script.yml",
+    "Sensor Health":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/sensor_health.yml",
+    "Service":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/service.yml",
+    "Snapshot":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/snapshot.yml",
+    "User Account":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/user_account.yml",
+    "Volume":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/volume.yml",
+    "Web Credential":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/web_credential.yml",
+    "Windows Registry":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/windows_registry.yml",
+    "WMI":"https://github.com/mitre-attack/attack-datasources/blob/main/contribution/wmi_object.yml"
+}
+
+def data_sources_link(data_sources):
+    """ Data source mapping for April release """
+
+    link = "<a target='_blank' href='{}'>{}</a>"
+
+    def replace_link(data_sources):
+        
+        updated_data_sources = []
+
+        for data_source in data_sources:
+            data_source_str = data_source
+            if ":" in data_source:
+                data_source_name = data_source.split(":")[0]
+                data_source_component = data_source.split(":")[1]
```

Comment:
You may run into issues here if you don't trim. data_source_component is likely to have a value `" component name"` after splitting on the `:` since data_source will usually have a space after the colon. Since you add a space on line 1033 before the data source component you'd end up with a double space.

This wouldn't be visible when rendered in HTML because HTML removes double whitespace (except when CSS is set in a specific way). Even so, still worth fixing just in case especially for anything automated that may be trawling the website (including our own search setup). Calling `strip` on the name and component may be sufficient.